pom.xml

```diff
@@ -122,6 +123,14 @@
 			<version>1.64</version>
 			<optional>true</optional>
 		</dependency>
+		
+		<!-- dependencies for Client Certificate Authentication -->
+		<dependency>
+			<groupId>org.bouncycastle</groupId>
+			<artifactId>bcpkix-jdk15on</artifactId>
+			<version>1.64</version>
```

Comment:
create a variable for the version and use it for both bcpkix-jdk15on and bcprov-jdk15on

Reply:
done.